zwave-classifier.js

```diff
@@ -70,6 +72,30 @@ const NOTIFICATION_SENSOR = {
     },
     valueListMap: [false, true],
   },
+  [NOTIFICATION_CO_DETECTOR]: {// 2
+    name: 'carbon_monoxide',
+    propertyName: 'carbon_monoxide',
+    propertyDescr: {
+      '@type': 'CarbonMonoxideProperty',
+      type: 'boolean',
+      label: 'Carbon Monoxide',
+      description: 'Carbon Monoxide (CO) Detector',
+      readOnly: true,
+    },
+    valueListMap: ['Clear', 'CO Detected'],
+  },
+  [NOTIFICATION_HEAT]: {// 4
+    name: 'heat',
```

Comment:
Lets call this overheat, since some sensors (probably not CO) have overheat and underheat alarms.